src/Symfony/Component/Console/Input/InputInterface.php

```diff
@@ -18,6 +18,9 @@
  * InputInterface is the interface implemented by all input classes.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @method string __toString() Returns a stringified representation of the args passed to the command.
+ *                             InputArguments MUST be escaped as well as the InputOption values passed to the command.
```

Comment:
I could probably link to the existing implementations for further clarifications? Or is this assumed to be enough?

Reply:
I am fine with it